client/state/posts/test/reducer.js

```diff
@@ -114,7 +147,7 @@ describe( 'reducer', () => {
 				}
 			} );
 		} );
-		it( 'never persists state because this is not implemented', () => {
+		it( 'never persists state', () => {
```

Comment:
Why do we not persist this? Or at the very least, we could repopulate the mapping from the set of loaded posts.